### PR TITLE
Apply a workaround for setting the default device orientation

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Please merge it after https://github.com/appium/appium-ios-simulator/pull/130 is published

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -657,11 +657,33 @@ class XCUITestDriver extends BaseDriver {
...
       connectHardwareKeyboard: !!this.opts.connectHardwareKeyboard,
       isHeadless: !!this.opts.isHeadless,
-    });
+    };
+
+    // This is to workaround XCTest bug about changing Simulator
+    // orientation is not synchronized to the actual window orientation
+    const orientation = this.opts.orientation;
+    if (_.isString(orientation)) {
+      switch (orientation.toUpperCase()) {
```
Comment (imurchie) [inline lib/driver.js]: This transform should be on line 668. Seems conceptually very strange to run it on something that is a constant.

**Comment by imurchie** [comment]
`appium-ios-simulator@2.3.0` published.

File: lib/driver.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/driver.js]: corrected

**Comment by imurchie** [comment]
Published in `appium-xcuitest-driver@2.51.0`.

**Pull request merged**